website_sale_qty/tests/test_js.py

```diff
@@ -0,0 +1,18 @@
+# # -*- coding: utf-8 -*-
+# # Copyright 2016 LasLabs Inc.
+# # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+#
+# from odoo.tests import HttpCase
```

Comment:
See [this diff](https://github.com/OCA/website/pull/329/files#diff-14507ac00444cf9322aaf5cf5981c5e9R8) for upgrading to v10